build.py

```diff
@@ -0,0 +1,37 @@
+import shutil
+import sass
```

Comment:
This is third party, don't group it with stdlib. `isort` can help you with this.

Reply:
excellent suggestion. thanks!